runtime/dali/src/weights/bonded_finance.rs

```diff
@@ -0,0 +1,57 @@
+
+//! Autogenerated weights for `bonded_finance`
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
+//! DATE: 2022-02-02, STEPS: `10`, REPEAT: 5, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("dali-dev"), DB CACHE: 128
+
+// Executed Command:
+// ./target/release/composable
+// benchmark
+// --chain=dali-dev
+// --execution=wasm
+// --wasm-execution=compiled
+// --pallet=bonded-finance
+// --extrinsic=*
+// --steps=10
+// --repeat=5
+// --raw
+// --output=./runtime/dali/src/weights
+
+#![cfg_attr(rustfmt, rustfmt_skip)]
```

Comment:
is this from your laptop or ci? just askin - cause only if CI has same spec as runtme machines than these will be good fit.

Reply:
CI will generate this file when adding the `needs-benchmarks` label :)